### ISSUE TITLE
Failed weekly report

### ISSUE THREAD

**Issue by geexup** [geexup/clockify-telegram-bot#3]
Weekly report crashed before image rendering:
```
Ooops TypeError: Cannot read property 'name' of undefined
    at _loop_1 (/Users/lexyudin/projects/clockify-autobot/src/scenes/report/reports/week.ts:65:37)
    at /Users/lexyudin/projects/clockify-autobot/src/scenes/report/reports/week.ts:135:25
    at step (/Users/lexyudin/projects/clockify-autobot/src/scenes/report/reports/week.ts:43:23)
    at Object.next (/Users/lexyudin/projects/clockify-autobot/src/scenes/report/reports/week.ts:24:53)
    at fulfilled (/Users/lexyudin/projects/clockify-autobot/src/scenes/report/reports/week.ts:15:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

Because the project variable was undefined
```
const project = projects.find((item) => item.id === projectId);
```

Project.get() returns only 50 projects. Projects from timesheet were not represented in this range.
```
const projects = await getClockify(ctx).workspaces(activeWorkspace.id).projects.get();
```

On Timesheet page, Clockify frontend uses the `entriesInRange` method which is not declared in [API](https://clockify.me/developers-api). Maybe this will work.

**Comment by geexup** [comment]
**UPDATE**: IT WORKS WITH Api-Key!

But it’s *really* weird.
All API is REST but this method uses POST to get data and has some strange url/path logic.

Soon I'll add this method to [geexup/clockify-api](https://github.com/geexup/clockify-api) and update dependencies 😈

**Comment by geexup** [comment]
**FIXED** in version 1.3.0
